test/helper.rb

```diff
@@ -15,36 +20,8 @@
   end
 end
 
-ENV['RACK_ENV'] = ENV['RAILS_ENV'] = 'test'
-
-trap 'TSTP' do
-  threads = Thread.list
-
-  puts
-  puts "=" * 80
-  puts "Received TSTP signal; printing all #{threads.count} thread backtraces."
-
-  threads.each do |thr|
-    description = thr == Thread.main ? "Main thread" : thr.inspect
-    puts
-    puts "#{description} backtrace: "
-    puts thr.backtrace.join("\n")
-  end
-
-  puts "=" * 80
-end
-
-require 'minitest/autorun'
-
 Sidekiq.logger.level = Logger::ERROR
 
-REDIS_URL = ENV['REDIS_URL'] || 'redis://localhost/15'
```

Comment:
I don't want the test suite to use the default Redis `localhost:6379/0`

Reply:
OK, I may return back `ENV['REDIS_URL'] ||= 'redis://localhost/15'`, that should be enough I guess.
Why `15`? Is it kind of virtual host in RabbitMQ?

Reply:
Don't want to be a dick, but we already have tests which rely on default Redis host ^_^
https://github.com/mperham/sidekiq/blob/master/test/test_redis_connection.rb#L84
https://github.com/mperham/sidekiq/blob/master/test/test_redis_connection.rb#L126

Reply:
Oops, yeah, I want people's default redis database to be untouched by Sidekiq's test suite.  We'll need to fix those tests (or make them read only)

Reply:
Already done this, Mike ^_^